src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -34,29 +35,41 @@
         </v-card-text>
       </template>
 
-      <!--  List Tables -->
-      <template v-else-if="!showSocietiesInfo">
-        <v-card-text class="">
+      <!-- Special case for societies -->
+      <template v-if="showSocietiesInfo">
```

Comment:
I simply moved this up from below.

(I think it's easier to understand "if true else everything else" instead of "if not true ...".)